src/api/config/routes/webui_routes.rb

```diff
@@ -401,9 +401,14 @@
       end
     end
 
+    ### /apidocs-old
+    get 'apidocs-old', to: redirect('/apidocs-old/index')
+    get 'apidocs-old/(index)' => 'webui/apidocs#index', as: 'apidocs_index'
+
+    # Redirections to be removed, just to keep the legacy routes reachable
     ### /apidocs
-    get 'apidocs', to: redirect('/apidocs/index')
-    get 'apidocs/(index)' => 'webui/apidocs#index', as: 'apidocs_index'
+    get 'apidocs', to: redirect('/apidocs-old/index')
+    get 'apidocs/(index)', to: redirect('apidocs-old/index')
```

Comment:
If I understand correctly, `/apidocs` will represent the new swagger-ui? If that is not the case then what will be the URL to reach swagger-ui? Maybe I'm missing something?

Reply:
`/apidocs` will represent the new swagger-ui, yes. But not yet. This plan doesn't perform all the tasks at once. This is the plan:
- First we create `/apidocs-old`, with the same contents of `/apidocs`
- Then we redirect `/apidocs` to `/apidocs-old`
At this point every user trying to access `/apidocs` will be redirected to `/apidocs-old`. No harm here. They will see the same page, with another path.

When we roll out the new documentation `/apidocs` will render the new documentation, with swagger-ui. Those redirections (from `/apidocs` to `/apidocs-old`) will be removed.

This workflow provides a constrained interval of time where users will be redirected to the definitive path. Later on, they will be presented the new interface. That is a less invasive option than to migrate everything at the same time.

Reply:
Ah okay, I see. This is an intermediate step.